### DISCUSSION TITLE
Please revert the sidebar change

### DISCUSSION THREAD

**Discussion by jlg-repo** [zen-browser/desktop#2606] [Ideas]
The thing that drew me to Zen browser and made it my default was the fact that I could manage screen space so efficiently by having things permanently open in the sidebar and then more important things open alongside it. However, the sidebar now pops out over the main window and thus is meant for just quickly doing something and then closing it. I don't really see what the point of this is and much prefer when it could sit right alongside the main window without interfering with it.

**Comment by jeanlucthumm** [comment]
Have you tried right clicking the sidebar and hitting "Enable Compact Mode" (as well as playing with the settings?)

**Reply by jlg-repo** [reply]
This just hid the tabs. What I'm looking to do is restore the pre-update behavior of the sidebar (not the toolbar on the side with the tabs), where you could have a mobile-like interface with mini tabs open on the left, while the main window was open alongside it, and the sidebar was not on top of the main window, because then it gets in the way.

**Reply by Tellinq** [reply]
> This just hid the tabs. What I'm looking to do is restore the pre-update behavior of the sidebar (not the toolbar on the side with the tabs), where you could have a mobile-like interface with mini tabs open on the left, while the main window was open alongside it, and the sidebar was not on top of the main window, because then it gets in the way.

The `-` button on Zen Sidebar should at least give back the functionality you are looking for. Should also persist through restarts (for now).

**Reply by jlg-repo** [reply]
> > This just hid the tabs. What I'm looking to do is restore the pre-update behavior of the sidebar (not the toolbar on the side with the tabs), where you could have a mobile-like interface with mini tabs open on the left, while the main window was open alongside it, and the sidebar was not on top of the main window, because then it gets in the way.
> 
> The `-` button on Zen Sidebar should at least give back the functionality you are looking for. Should also persist through restarts (for now).

Lovely, this is exactly what I wanted it to do. Please please don't take this out!
